### DISCUSSION TITLE
Cüzdan Puan İşlemi

### DISCUSSION THREAD

**Discussion by ridvanbesli** [IBAX-io/go-ibax#1834] [Soru Cevap - Türkçe] [answered]
Arkadaşlar puanı şuanda cüzdana token olarak nasıl aktarabiliyoruz? İşlemleri Şubat ayında yapacaktık diye hatırlıyorum

**Comment by Yusufknk** [comment]
Airdrop 1 Şubat'ta sondru. Ön satıştan sonra dağıtım olacak.

**Comment by yolandadadada** [comment, marked as answer]
It is in the process of being liquidated and will be redeemed in the second quarter of 2022.
